src/commands/secrets/download.ts

```diff
@@ -0,0 +1,72 @@
+import fs from 'fs-extra';
+import yaml from 'js-yaml';
+import path from 'path';
+import tmp from 'tmp';
+import untildify from 'untildify';
+import AccountUtils from '../../architect/account/account.utils';
+import { EnvironmentUtils } from '../../architect/environment/environment.utils';
+import Command from '../../base-command';
+import { SecretsDict } from '../../dependency-manager/secrets/type';
+
+tmp.setGracefulCleanup();
+
+export default class SecretsDownload extends Command {
```

Comment:
This should extend `BaseCommand` instead of `Command` otherwise we lose out on all our custom logic.